### PR TITLE
Wiki conversion to 4.1

### PR DESCRIPTION
**Pull request by Hwurzburg**
Moving Copter and Rover and common items out of upcoming into main wiki

### CODE REVIEW

File: common/source/docs/common-advanced-configuration.rst [hunk 1]
```diff
@@ -62,13 +62,13 @@ tuning options for the vehicle.
     Ground Control Station Only Operation <common-gcs-only-operation>
 [site wiki="copter"]
     Ground Effect Compensation <ground-effect-compensation>
-    In-Flight FFT Configuration <common-imu-fft>
```
Comment (rmackay9) [inline common/source/docs/common-advanced-configuration.rst:63]: Hi, is it "FFT" or "FTT"?

**Comment by rmackay9** [comment]
This looks OK to me.  I guess we should merge this now because we've started beta testing?  If we do it will perhaps cause some confusion because users of stable will find some new features don't work... but beta testers are more likely to find these changes.

I guess another option would be to more explicitly point beta testers to the "New Features" section of the wiki.

Anyway, I'm happy with this being merged any time.

File: common/source/docs/common-advanced-configuration.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-advanced-configuration.rst:63]: Fast Fourier Transform

**Comment by Hwurzburg** [comment]
Still a lot more work to do, but am going to merge this now and continue working on existing PRs and issue list

**Pull request merged**